app/static/md/random.md

```diff
@@ -19,7 +19,7 @@ ask patches [
 Things to keep in mind when using `random`:
 
 * Always remember that `random` will give us numbers within the range from 0 to N - 1, instead of 1 to N. For example, if we run `random 3`, we may get 0, 1, or 2, *but not 3*. 
```

Comment:
```suggestion
* Always remember that `random` will give us numbers within the range from 0 to N - 1, instead of 1 to N. For example, if we run `random 3`, we may get 0, 1, or 2, *but not 3*. In math and computer science language, we would say that the `random` primitive produces numbers in the range 0 to N *exclusive*, meaning that the number N itself is *excluded* from the possible set of values. 
```